networking/ovn-kubernetes-network-provider/about-ovn-kubernetes.adoc

```diff
@@ -0,0 +1,26 @@
+[id="learn-about-ovn-kubernetes"]
+= Learn about the OVN-Kubernetes Pod network provider
+include::modules/common-attributes.adoc[]
+:context: learn-about-ovn-kubernetes
+
+toc::[]
+
+[id="ovn-kubernetes-metrics"]
+== Exposed metrics
+
+The OVN-Kubernetes network provider exposes certain metrics for use by the Prometheus-based {product-title} cluster monitoring stack.
+
+// openshift/ovn-kubernetes => go-controller/pkg/ovn/metrics.go
+
+.Metrics exposed by OVN-Kubernetes
```

Comment:
Hi @JacobTanenbaum,

I added this table for metrics; Are these correct? Am I missing any? What do values for `nb_e2e_timestamp` mean? Is it a delta value or is it the actual (unix?) timestamp? What range of values would signify a problem with OVN-Kubernetes?

Thanks!